scripts/legacy_expression_tool.py

```diff
@@ -0,0 +1,394 @@
+
+fiware_service = {'$regex': '.*'}
+if args['service']:
+    fiware_service = args['service']
+    if debug==True:
+        print('Filtering by service: ' + fiware_service)
+
+fiware_servicepath = {'$regex': '.*'}
+if args['servicepath']:
+    fiware_servicepath = args['servicepath']
+    if debug==True:
+        print('Filtering by servicepath: ' + fiware_servicepath)
+
+filter_device_id = ''
+if args['filterdevices']:
+    filter_device_id = args['filterdevices']
+    print('Filtering by device ID: ' + filter_device_id)
+
+if filter_device_id != '':
+    filter['$and'].append({'id': {'$regex': filter_device_id}})
+
+args['translation'] = 'translation.json'
+if args['translation'] != None:
+    with open(args['translation']) as f:
+        translation_legacy_expressions = json.load(f)
+
+mongodb_db = args['database']
+mongodb_collection = args['collection']
+
+_regex_legacy_expression = '\\${.*(@)'
+
+# Create a client instance of the MongoClient class
+client = MongoClient('mongodb://'+host+':'+port+'/') # Create a client instance of the MongoClient class
+if debug==True:
+    print('Connected to mongodb://'+host+':'+port+'/')
+
+# Create a filter for the query
+filter = {
+    '$and':[
+        {'$or':[
+            {'active': {'$elemMatch': {'expression': {'$regex': _regex_legacy_expression}}}},
+            {'active': {'$elemMatch': {'entity_name': {'$regex': _regex_legacy_expression}}}},
+            {'active': {'$elemMatch': {'reverse': {'$elemMatch': {'expression':{'$regex': _regex_legacy_expression}}}}}},
+            {'attributes': {'$elemMatch': {'expression': {'$regex': _regex_legacy_expression}}}},                                # grups
+            {'attributes': {'$elemMatch': {'entity_name': {'$regex': _regex_legacy_expression}}}},                               # grups
+            {'attributes': {'$elemMatch': {'reverse': {'$elemMatch': {'expression':{'$regex': _regex_legacy_expression}}}}}},    # grups
+            {'commands': {'$elemMatch': {'expression': {'$regex': _regex_legacy_expression}}}},
+            {'endpoint': {'$regex': _regex_legacy_expression}},
+            {'entityNameExp': {'$regex': _regex_legacy_expression}},
+            {'explicitAttrs': {'$regex': _regex_legacy_expression}},
+            ]
+        },
+        {'service': fiware_service},
+        {'subservice': fiware_servicepath},
+        # {'id': {'$regex': '.*'}},
+        ]
+}
+
+expressionlanguage = 'ignore'
+if args['expressionlanguage'] == 'delete':
+    filter['$and'][0]['$or'].append({'expressionLanguage':{'$exists': True}})
+    expressionlanguage = 'delete'
+elif args['expressionlanguage'] == 'jexl':
+    filter['$and'][0]['$or'].append({'expressionLanguage':{'$exists': True}})
+    expressionlanguage = 'jexl'
+elif args['expressionlanguage'] == 'ignore':
+    expressionlanguage = 'ignore'
+
+
+if debug==True:
+    print('Filter: ' + str(filter))
+
+# Execute find query
+result_cursor = client[str(mongodb_db)][str(mongodb_collection)].find(
```

Comment:
Why don't just

```suggestion
result_cursor = client[mongodb_db][mongodb_collection].find(
```

?